pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -58,10 +58,10 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 		// some oddities on systems with more than 64 processors.
 		// Refer https://msdn.microsoft.com/en-us/library/windows/desktop/dd405503(v=vs.85).aspx.
 		cpuMaximum := 10000 * cpuLimit.MilliValue() / int64(sysinfo.NumCPU()) / 1000
-		if isolatedByHyperv {
-			cpuCount := int64(cpuLimit.MilliValue()+999) / 1000
-			wc.Resources.CpuCount = cpuCount
+		cpuCount := int64(cpuLimit.MilliValue()+999) / 1000
+		wc.Resources.CpuCount = cpuCount
```

Comment:
AFAIK, CpuCount is only supported by HyperV isolation.

cc @PatrickLang

Reply:
Just confirmed, I came to the test environment based on the process isolation level of windows container instead of hyper-v

Reply:
here's order of precedence in Moby code
https://github.com/moby/moby/blob/1bd184a4c291e4f60629e2cc279216f8f40495f3/daemon/daemon_windows.go#L88

Reply:
I've seen docker's processing order before. It's worth mentioning that there is no limit on the `--cpus` parameter in docker. This parameter seems to be able to more accurately limit the use of CPUs in the windows container.

Reply:
@PatrickLang   The windows container parameter `cpuMaximum` is assigned through the` NanoCPUs` parameter. Docker does not provide an interface, and directly assigns `cpuMaximum`. Therefore, we should only assign the parameter `NanoCPUs`, and then call docker.

Here's precedence in Moby code
https://github.com/moby/moby/blob/2d467dc8d00790da8f4f8d1c0ab3ed7811b7958f/daemon/oci_windows.go#L413-L432